_rules/SC1-1-1-image-has-name.md

```diff
@@ -25,8 +25,6 @@ The rule applies to HTML `img` elements or any HTML element with the [semantic r
 
 Each target element has an [accessible name](#accessible-name) that is [non-empty](#non-empty) or is marked as [decorative](#decorative).
 
-**NOTE**: An `img` element can be marked as decorative, by using either `role="presentation"`, `role="none"` or an empty alt attribute `alt=""`.
```

Comment:
Why did you remove this note?

Reply:
@WilcoFiers I added this information to the [decorative](https://github.com/auto-wcag/auto-wcag/blob/Brynanders-patch-1/pages/algorithms/decorative.md) definition and therefore considered the note duplication of the definition.